dataland-frontend/src/components/forms/CreateEuTaxonomyForFinancials.vue

```diff
@@ -546,6 +580,42 @@ export default defineComponent({
       this.onThisPageLinks = [...new Set(this.onThisPageLinksStart.concat(this.selectedFinancialServiceOptions))];
     },
 
+    /**
+     * Converts the kpisSection data to a request-friendly object
+     * @param kpiSections Kpi Section data
+     * @returns ObjectType
+     */
+    makeEligibilityKpis(kpiSections: ObjectType) {
```

Comment:
Again, its not all about the eligibility kpis. I propose `buildKpis`.

Reply:
Or `convert...`

Reply:
Done.

Reply:
There is still the word `eligibility` inside the name of the function